runtime/lua/vim/secure.lua

```diff
@@ -1,5 +1,6 @@
 local M = {}
 
+--- @private
```

Comment:
This goes for all annotations, but no space between `-` and `@`

Reply:
Resolved in 3e7837abc58078f82b69a551fc829bc2adfeedb7